eng/versioning/version_client.txt

```diff
@@ -48,7 +48,7 @@ com.azure:azure-test-watcher;1.0.0-beta.1;1.0.0-beta.1
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
-
+unreleased_com.azure:azure-core-amqp;1.2.0-beta.1
 # Released Beta dependencies: Copy the entry from above, prepend "beta_", remove the current
```

Comment:
Add new line. This is lost in the paragraph of text.

Reply:
Since this affects Event Hubs too, the Event Hubs dependency should be updated and the live tests run.